[2024-06-21T07:37:02.171+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-06-21T07:37:02.188+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow.load_pitStops_task scheduled__2024-06-20T00:00:00+00:00 [queued]>
[2024-06-21T07:37:02.195+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: f1_data_pipeline_taskflow.load_pitStops_task scheduled__2024-06-20T00:00:00+00:00 [queued]>
[2024-06-21T07:37:02.196+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-06-21T07:37:02.206+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): load_pitStops_task> on 2024-06-20 00:00:00+00:00
[2024-06-21T07:37:02.212+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=505) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-06-21T07:37:02.214+0000] {standard_task_runner.py:63} INFO - Started process 519 to run task
[2024-06-21T07:37:02.214+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'f1_data_pipeline_taskflow', 'load_pitStops_task', 'scheduled__2024-06-20T00:00:00+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/etl_dag.py', '--cfg-path', '/tmp/tmpyepg0uam']
[2024-06-21T07:37:02.215+0000] {standard_task_runner.py:91} INFO - Job 29: Subtask load_pitStops_task
[2024-06-21T07:37:02.257+0000] {task_command.py:426} INFO - Running <TaskInstance: f1_data_pipeline_taskflow.load_pitStops_task scheduled__2024-06-20T00:00:00+00:00 [running]> on host f9e8c8c49316
[2024-06-21T07:37:02.378+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='f1_data_pipeline_taskflow' AIRFLOW_CTX_TASK_ID='load_pitStops_task' AIRFLOW_CTX_EXECUTION_DATE='2024-06-20T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2024-06-20T00:00:00+00:00'
[2024-06-21T07:37:02.378+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-06-21T07:37:02.379+0000] {crypto.py:82} WARNING - empty cryptography key - values will not be stored encrypted.
[2024-06-21T07:37:02.380+0000] {base.py:84} INFO - Using connection ID 'sourcedb_connection' for task execution.
[2024-06-21T07:37:02.519+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-06-21T07:37:02.519+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-06-21T07:37:02.526+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=f1_data_pipeline_taskflow, task_id=load_pitStops_task, run_id=scheduled__2024-06-20T00:00:00+00:00, execution_date=20240620T000000, start_date=20240621T073702, end_date=20240621T073702
[2024-06-21T07:37:02.547+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-06-21T07:37:02.565+0000] {taskinstance.py:3498} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-06-21T07:37:02.566+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
